src/bp/core/api.ts

```diff
@@ -100,6 +101,9 @@ const config = (moduleLoader: ModuleLoader, configProfider: ConfigProvider): typ
     },
     getBotpressConfig(): Promise<any> {
       return configProfider.getBotpressConfig()
+    },
+    mergeBotConfig(botId: string, partialConfig: PartialDeep<sdk.BotConfig>, ignoreLock?: boolean): Promise<any> {
+      return configProfider.mergeBotConfig(botId, partialConfig, ignoreLock)
     }
```

Comment:
When editing something in the SDK, try to use the bind method instead, because if we change parameters, there is only one place that needs to be updated, ex:
 mergeBotConfig: configProvider.mergeBotConfig.bind(configProvider),

(also just saw the typo "profider")

Reply:
Done in https://github.com/botpress/botpress/pull/2417/commits/edf6c4f55749f4da4b1da1921be6b45f8ed87f91 and https://github.com/botpress/botpress/pull/2417/commits/f4769bb1ad09903958b2b2fe7fd445774cc82eb2